assets/initial_algorithms.json

```
@@ -0,0 +1,136 @@
+[
+    {
+        "label": "dibbs-basic",
+        "description": "The DIBBs default algorithm based on field experimentation and statistical analysis, this is a two-pass algorithm combining geographical and personal information to maximize linkage quality while minimizing false positives",
```

I think the DIBBs Enhanced algorithm should be characterized as the default. What do you think?

From my understanding, the DIBBs Basic algorithm was created to offer users a simpler, more explainable option, but because it performs more poorly in practice because it is deterministic we opt for the DIBBs Enhanced algorithm. Probabilistic RL algorithms like the DIBBs Enhanced also seem more aligned with NBS' understanding & implementation of RL (e.g., log odds).

Also, I think the word 'deterministic' should be included in this DIBBs Basic description. As is, this description also holds for the DIBBs Enhanced algorithm, and I think the two could be differentiated more clearly. Happy to follow up with a suggestion.

---

From the DIBBs teams definitions, I was leaving basic as the default because a) they said it should work for most and b) log odds needs to be trained on a STLTs data first to correctly calculate the values.

However, if you and/or the DIBBs team think we should switch the default, please feel free to make the switch.

As for the description, sure just click the "add a suggestion" button and write in a new one if you think we should change.

---

Ah that makes sense. If it's in the DIBBs team's definitions, let's keep it that way now til further notice. I haven't discussed with them explicitly whether DIBBs Enhanced should be the default, so that may have just been my assumption.

Here's a suggestion
```suggestion
        "description": "The DIBBs Default Algorithm. Based on field experimentation and statistical analysis, this deterministic two-pass algorithm combines geographical and personal information to maximize linkage quality while minimizing false positives",
```